src/rpc_client/src/connector_client.rs

```diff
@@ -272,6 +272,7 @@ impl ConnectorClient {
         let (handle, first_rsp) = SinkWriterStreamHandle::initialize(
             SinkWriterStreamRequest {
                 request: Some(SinkRequest::Start(StartSink {
+                    payload_schema: sink_param.table_schema.clone(),
```

Comment:
We should make the `start_sink_writer_stream` of both connector clients to be consistent. Change to the same parameter list as the `start_sink_writer_stream` of `EmbeddedConnectorClient`, which passes the separate payload schema instead of using the schema in sink param.